src/components/topic/topic.jsx

```diff
@@ -76,7 +76,7 @@ export default class Topic extends React.Component {
 
                 <section className={ `${block}__vote` }>
                     <div className={ `${block}__title ${block}__title--vote` }>
-                        Place Your Vote
+                        {topic.locked ? 'Voting Ended' : 'Place Your Vote'}
```

Comment:
Did you mean `disabled`? By the look of it you are checking if its locked 
```suggestion
                        {topic.locked ? 'Voting not allowed' : 'Place Your Vote'}
```

Reply:
Done